sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -98,20 +96,27 @@ def set_workload_id(self, workload_id):
 
     def set_requester_id(self, requester_id):
         """Set requesterId work order parameter."""
+        if requester_id is None or \
+                not is_valid_hex_str(requester_id):
+            raise ValueError("Empty or Invalid data \
+format for requester_id")
         self.params_obj["requesterId"] = requester_id
 
     def set_worker_encryption_key(self, worker_encryption_key):
         """Set workerEncryptionKey work order parameter."""
-        try:
-            self.params_obj["workerEncryptionKey"] = \
-                worker_encryption_key.encode("UTF-8").hex()
-        except Exception as err:
-            raise TypeError("Worker Encryption Key not valid")
+        if worker_encryption_key is None or \
+                not isinstance(worker_encryption_key, str) or \
+                not worker_encryption_key.strip():
+            raise ValueError("Empty or Invalid data \
+format for workerEncryptionKey")
```

Comment:
Use this form instead (similar to C++) to continue strings:

```
raise ValueError("Empty or Invalid data "
    "format for workerEncryptionKey")
```

Reply:
done